### DISCUSSION TITLE
Phobos Versioning and Distribution

### DISCUSSION THREAD

**Discussion by pbackus** [dlang/PhobosV3-Design#10] [General]
From [the forums][1]:

> After that we had a discussion about how to distribute Phobos. This mostly centered on what release cadence to use. I argued for linking the Phobos version to the edition release schedule. I think this is sensible and makes it easier for people to reason about which compiler/library pairing they are using. Walter was fine with that, but he does not want to use the "Edition" language to describe Phobos releases. I think this makes sense as Phobos doesn't really have editions, but following SemVer, it would have yearly major releases. This would mean that new features are released once per year, and bugfixes would be released on whatever the fast-cadence schedule is for DMD in between Yearly Edition roll-ups.

[1]: https://forum.dlang.org/post/hqliwbdrimuzzworupdt@forum.dlang.org

**Comment by atilaneves** [comment]
Having thought about this yesterday, I agree that new Phobos versions should be released at the same time as a new edition.

**Reply by LightBender** [reply]
This is awesome to hear, I wrote up a PR for the design doc to add this language: https://github.com/LightBender/PhobosV3-Design/pull/19

**Comment by LightBender** [comment]
This has been approved and merged.
